flake8_trio/visitors/visitor105.py

```diff
@@ -0,0 +1,47 @@
+"""TRIO105: library async function must be immediately awaited."""
```

Comment:
Entirely unchanged, moved out from `visitors.py` in preparation for getting a bunch of extra functionality.